.eslintrc.js

```diff
@@ -127,14 +127,15 @@ module.exports = {
         'packages/api-server/src/**',
         'packages/cli/src/**',
         'packages/core/config/**',
-        'packages/create-redwood-app/src/create-redwood-app.js',
```

Comment:
You might have made this change accidentally @cannikin - not sure if we want this change? I don't know either...

Reply:
`create-redwood-app.js` was the only file in the directory, but now I've added `sendTelemetry.js` in there as well. So I changed to a glob to just scan all files in there with `*.js`. I wouldn't have cared, but it was throwing some errors after I added `sendTelemetry.js` 🤷‍♂️